octobot_trading/data/order.py

```diff
@@ -69,6 +77,9 @@ def __init__(self, trader):
     def get_name(cls):
         return cls.__name__
 
+    def get_logger_name(self):
+        return f"{self.get_name()} | {self.order_id}"
```

Comment:
:+1:

Reply:
Should we save this in an attribute to avoid re-computing it all the time on logging ?